nixos/modules/security/wrappers/wrapper.nix

```diff
@@ -16,6 +16,7 @@ stdenv.mkDerivation {
   dontStrip = debug;
   installPhase = ''
     mkdir -p $out/bin
-    $CC $CFLAGS ${./wrapper.c} -o $out/bin/security-wrapper
+    cp ${glibcSrc}/sysdeps/generic/unsecvars.h .
+    $CC $CFLAGS ${./wrapper.c} -I. -o $out/bin/security-wrapper
```

Comment:
```suggestion
    $CC $CFLAGS -Os ${./wrapper.c} -I. -o $out/bin/security-wrapper
```

Reply:
The wrappers come out to 63432 bytes exactly, both with and without `-Os`.